gains:
  shoulder_pan_joint: {p: 1, d: 0, i: 0, i_clamp: 1}
  shoulder_lift_joint: {p: 1, d: 0, i: 0, i_clamp: 1}
  elbow_joint: {p: 1, d: 1, i: 0, i_clamp: 1}
  wrist_1_joint: {p: 1, d: 0, i: 0, i_clamp: 1}
  wrist_2_joint: {p: 1, d: 0, i: 0, i_clamp: 1}
  wrist_3_joint: {p: 1, d: 0, i: 0, i_clamp: 1}
